src/amo/components/HeroRecommendation/index.js

```diff
@@ -242,7 +242,9 @@ export class HeroRecommendationBase extends React.Component<InternalProps> {
           <div className="HeroRecommendation-title-text">
             {titleText || <LoadingText width={20} />}
           </div>
-          {![LINE, RECOMMENDED].includes(promotedCategory) && !loading ? (
+          {![LINE, RECOMMENDED].some((item) =>
```

Comment:
Nit: totally not a big deal in this case but I wanted to call out a few good general programming topics, especially with mapping arrays.

1) Cheap compute should go before (it was like that before) expensive compute. so if `!loading` evaluates to `false` we can skip the some/includes check. Not a big deal hear as the lists are tiny, but good to think about since theoretically promotedCategories is an arbitrarily large array.

2) negations on array methods can be hard to comprehend, and normally can be inverted.

```js
![LINE, RECOMMENDED].some((item) => promotedCategories.includes(item))
// this should be equivalent to
promotedCategories.every((cat) => [LINE, RECOMMEND].includes(cat))
```

3) nested array lookups should be avoided if possible. These are small arrays but it's a good habit to form so when it is an issue, you don't accidentally do it (I did  it last week so I've felt the pain)
4) It is good to name variables when you have sets of logical definition like in this case we are matching if the set of categories on the addon is a subset of the categories that we recommend, so you could set this to a variable and let the render body use the variable
```jsx
// probably unecessary to use a set, but is technically faster
const RECOMMENDED_CATEGORIES = Set([LINE, RECOMMENDED])
const isRecommendedAddon = !isLoading && promotedCategories.every((cat) => RECOMMENDED_CATEGORIES.has(cat));

{isRecommendedAddon ? <Foo /> : <Bar />}
```